packages/prop-house-backend/src/auction-watcher/auction-watcher.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { Auction } from 'src/auction/auction.entity';
+import { AuctionsService } from 'src/auction/auctions.service';
+import { AuctionWatcherService } from './auction-watcher.service';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Auction])],
+  controllers: [],
+  providers: [AuctionsService, AuctionWatcherService],
+  exports: [],
+})
+export class AuctionWatcherModule {}
```

Comment:
Doesn't matter too much either way, but created the `TasksModule` to house cron/interval tasks

Reply:
Was hoping to avoid tangling too many dependencies together so will keep them separate for now.